Sources/Views/NewEpisodePage.swift

```diff
@@ -228,21 +228,10 @@ private func collectionHeaderRow(
     ],
     .gridColumn(
       sizes: [.mobile: 12],
-      attributes: [
-        .class([
-          Class.padding([.mobile: [.left: 1]]),
-        ])
-      ],
       .h6(
         attributes: [
           .class([
-            Class.pf.colors.fg.gray650,
-            Class.typeScale([.mobile: .r0_75]),
-            Class.type.lineHeight(1),
-            Class.type.caps,
-            Class.type.normal,
-            Class.padding([.mobile: [.all: 0]]),
-            Class.margin([.mobile: [.leftRight: 0, .top: 0, .bottom: 1]])
+            sidebarShoutTitleClass
```

Comment:
these classes were repeated for two titles, so extracted to something that could be shared